client/config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -5,7 +5,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.2.5
-    api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/260"
+    api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/139"
```

Comment:
Can you change the controller-gen version from v0.2.5 to 0.3.0?
controller-gen.kubebuilder.io/version: v0.3.0

Reply:
@xing-yang in my enviorment whenever i run the script for update-crd.sh its automatically takes the 0.2.5 version.However i manually update this version to 0.3.0

Reply:
You can't manually change this version.  This has to be automatically generated.  It means you have controller-gen 0.2.5 installed on your setup.

Check `controller-gen --version`

Reply:
@xing-yang yes, after updating the version in my enviorment it is working fine now when i run the script i got the expected output. The commit running fine